lib/auto-languageclient.js

```diff
@@ -251,23 +261,36 @@ export default class AutoLanguageClient {
     };
     this.postInitialization(newServer);
     connection.initialized();
+
+    let configurationKey = this.getRootConfigurationKey();
+    if (configurationKey) {
+      this._disposable.add(
+        atom.config.observe(configurationKey, config => {
+          connection.didChangeConfiguration({
+            settings: this.filterConfigurationObject(config || {})
+          })
+        }))
+    }
+
     this.startExclusiveAdapters(newServer);
     return newServer;
   }
 
   captureServerErrors(childProcess: child_process$ChildProcess, projectPath: string): void {
     childProcess.on('error', err => this.handleSpawnFailure(err));
     childProcess.on('exit', (code, signal) => this.logger.debug(`exit: code ${code} signal ${signal}`));
-    childProcess.stderr.setEncoding('utf8');
-    childProcess.stderr.on('data', (chunk: Buffer) => {
-      const errorString = chunk.toString();
-      this.handleServerStderr(errorString, projectPath);
-      // Keep the last 5 lines for packages to use in messages
-      this.processStdErr = (this.processStdErr + errorString)
-        .split('\n')
-        .slice(-5)
-        .join('\n');
-    });
+    if (childProcess.stderr) {
```

Comment:
This was needed because of the way I'm launching my language server in ide-powershell.  I don't actually use ChildProcess for this, I use node-pty instead.  As far as I know I can't get an actual ChildProcess object from there so this bit of code needed a guard to avoid trouble with stdio streams not being available.

Reply:
Decided to drop this change and create my own "fake" ChildProcess which conforms to what atom-languageclient expects.